tests/tests/cpld_control_c192_p8

```diff
@@ -56,6 +56,7 @@ export LIST_FILES="sfcf036.tile1.nc \
 
 export_fv3
 export_cpl
+export WLCLK=48
```

Comment:
Why do you need to increase the WLCLK time?

Reply:
This test needs about 43 minutes to run on Jet.

Reply:
Since this is for weekly test. We will remove this line in next PR and have a separate fix in weekly test.